guides/common/modules/proc_assigning-content-view-environments-to-an-activation-key-by-ID.adoc

```diff
@@ -0,0 +1,15 @@
+[id="assigning-content-view-environments-to-an-activation-key-by-ID"]
+= Assigning content view environments to an activation key by ID
+
+To assign content view environments to an activation key, determine the IDs of the content view environments you want to assign.
+
+.CLI procedure
+. Use `hammer content-view-environment list` to find and note the IDs you want to use.
+. Assign the selected content view environments to an activation key using their IDs:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer activation-key update \
```

Comment:
```suggestion
$ hammer activation-key update \
```

Reply:
I think you need to identify the activation key somehow - an option missing, such as AK ID?

Reply:
Yes, as @vijaysawant said here --> https://github.com/theforeman/foreman-documentation/pull/3609#pullrequestreview-2630267866